### PR TITLE
osa- skip-tests determinize-prow-jobs jobs invalidates skip regexp

### PR DESCRIPTION
**Pull request by mjudeikis**
when running it invaldiates regexp and tests are not skipped 
`docker run -it -v $(pwd)/jobs:/jobs:z registry.svc.ci.openshift.org/ci/determinize-prow-jobs:latest --prow-jobs-dir /jobs`

### CODE REVIEW

**Comment by mjudeikis** [comment]
related to this: https://github.com/openshift/ci-operator/issues/150

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml [hunk 1]
```diff
@@ -17,9 +17,11 @@ parameters:
 - name: CLUSTER_TYPE
   value: "azure"
   required: true
+- name: TEST_SUITE
```
Comment (wking) [inline ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml]: Why are you addin the `TEST_SUITE*` back?  They were intentionally removed in #1261.

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml [hunk 2]
```diff
@@ -122,17 +124,17 @@ objects:
...
         source /tmp/shared/overrides
         set -x
-        if [[ -n "${TEST_FOCUS}" ]]; then
+       if [[ -n "${TEST_SUITE}" || -n "${TEST_FOCUS}" ]]; then
           ginkgo -v -noColor -nodes=30 $( which extended.test ) -- \
-            -ginkgo.focus="${TEST_FOCUS}" -ginkgo.skip="${TEST_SKIP}" \
+            -suite "${TEST_SUITE}" -ginkgo.focus="${TEST_FOCUS}" -ginkgo.skip="${TEST_SKIP}" \
             -e2e-output-dir /tmp/artifacts -report-dir /tmp/artifacts/junit \
-            -test.timeout=2h || rc=$?
+            -test.timeout=2h ${PROVIDER_ARGS-} || rc=$?
```
Comment (wking) [inline ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml]: Why `PROVIDR_ARGS`?  If you want to consume it, you should probably set it.  For example:

```console
$ git grep PROVIDER_ARGS ci-operator/templates/cluster-launch-installer-e2e.yaml
ci-operator/templates/cluster-launch-installer-e2e.yaml:          export PROVIDER_ARGS='-provider
gce -gce-zone=us-east1-c -gce-project=openshift-gce-devel-ci'
ci-operator/templates/cluster-launch-installer-e2e.yaml:          export PROVIDER_ARGS="-provider
aws -gce-zone=${region}"
ci-operator/templates/cluster-launch-installer-e2e.yaml:            -test.timeout=10m ${PROVIDER_
RGS-} || rc=$?
ci-operator/templates/cluster-launch-installer-e2e.yaml:            -test.timeout=20m ${PROVIDER_
RGS-} || rc=$?
```

But if you don't have providers to distinguish between, it seems simpler to just remove `PROVIDER_ARGS` entirely.

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-presubmits.yaml [hunk 3]
```diff
@@ -206,19 +206,7 @@ presubmits:
...
-            report failed soon after an annotated objects has failed)|(s2i build with
-            a root user image should create a root build and pass with a privileged
-            SCC)).*
+          value: .*((The HAProxy router should set Forwarded headers appropriately)|(oc new-app should succeed with a --name of 58 characters)|(process valueFrom in build strategy environment variables should successfully resolve valueFrom in docker build environment variables)|(prune builds based on settings in the buildconfig should prune failed builds based on the failedBuildsHistoryLimit setting)|(Prometheus when installed to the cluster should start and expose a secured proxy and unsecured metrics)|(Prometheus when installed to the cluster should start and expose a secured proxy and verify build metrics)|(templateservicebroker)|(deploymentconfigs with multiple image change triggers [Conformance] should run a successful deployment with multiple triggers)|(templateinstance readiness test should report failed soon after an annotated objects has failed)|(s2i build with a root user image should create a root build and pass with a privileged SCC)).*
```
Comment (wking) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-presubmits.yaml]: >I strugled with getting either sane wrapping or no wrapping through CI, and ended up [giving up][1].  You unwrapped line here is hitting [that same issue][2].

[1]: https://github.com/openshift/release/pull/1341#issuecomment-418466429
[2]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/1520/pull-ci-openshift-release-master-ordered-prow-config/544/build-log.txt

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml (hunk 1 above)
Comment (mjudeikis) [reply ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml]: I was not aware of this.

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml (hunk 2 above)
Comment (wking) [reply ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml]: > Azure dont use it, so we remove it so not to confuse

So remove your addition here?

Comment (mjudeikis) [reply ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml]: done

File: ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml (hunk 1 above)
Comment (mjudeikis) [reply ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml]: done

**Comment by mjudeikis** [comment]
@kargakis ready to give it a second try

**Pull request merged**